docker-compose.yaml

```diff
@@ -5,6 +5,7 @@ services:
   caddy:
     container_name: caddy
     image: caddy:2-alpine
+    restart: always
```

Comment:
I think it would be better to use `restart: on-failure` instead of `always` because `always` will restart the container for everything that could stop the caddy container, even if the container exit for something unrelated to a failure.

Due to the fact that we don't know every single setup of searx-docker on the planet I would prefer to use `on-failure` because if caddy encounters an error then we are sure that the user would want to have an automatic restart of the container.